### ISSUE TITLE
VER command wrongly claims 0.80 is alpha

### ISSUE THREAD

**Issue by FeralChild64** [dosbox-staging/dosbox-staging#2158]
### Are you using the latest Dosbox-Staging Version?

- [X] I have checked releases and am using the latest release.

### Different version than latest?

0.80.0

### What Operating System are you using?

Linux x86_64

### If Other OS, please describe

Gentoo

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

Command VER wrongly claims 0.80 is an alpha.

### Steps to reproduce the behaviour.

1. Type VER from the command line
2. It says "DOSBox Staging version 0.80.0-alpha"

### Code of Conduct & Contributing Guidelines

- [X] Yes, I agree.

**Comment by kcgen** [comment]
@FeralChild64 , can you try a fresh clone + full tag pull, and also from the release sources tarball?

### fresh clone + full tag pull

``` shell
cd /dev/shm/
git clone https://github.com/dosbox-staging/dosbox-staging
cd dosbox-staging
git fetch --all --tags
VER=v0.80.0
git checkout $VER
meson setup build/$VER-release
meson compile -C build/$VER-release
./build/$VER-release/dosbox -c ver
```

### release sources tarball

``` shell
VER=v0.80.0
tarball=https://github.com/dosbox-staging/dosbox-staging/archive/refs/tags/$VER.tar.gz

cd /dev/shm/
wget -O - "$tarball" | tar zx
cd dosbox-staging-$VER
meson setup build/$VER-release
meson compile -C build/$VER-release
./build/$VER-release/dosbox -c ver
```

**Comment by FeralChild64** [comment]
Build from clone & tag: `meson setup build/$TAG-release` gives

```
The Meson build system
Version: 0.63.3
Source dir: /dev/shm/dosbox-staging
Build dir: /dev/shm/dosbox-staging/build/v0.80.0-release
Build type: native build
Project name: dosbox-staging
Project version: 0.80.0

meson.build:1:0: ERROR: Could not invoke sanity test executable: [Errno 13] Permission denied: '/dev/shm/dosbox-staging/build/v0.80.0-release/meson-private/sanitycheckc.exe'.

A full log can be found at /dev/shm/dosbox-staging/build/v0.80.0-release/meson-logs/meson-log.txt
```

Same for build from release sources and `meson setup build/$VER-release`.

**Comment by kcgen** [comment]
> Could not invoke sanity test executable

My guess is your ramdrive /dev/shm is mounted with `noexec` permissions, blocking the launch of any executable residing on that filesystem (it's become a security trend to block exec; maybe you can drop the noexec mount option for it; which would give you a pile of space for use for experiments like this that never touch your host filesystem). 

Alternatively, this is easy enough to work around,  if you want to runs those commands inside your home directory, say `~/testing/`.

**Comment by FeralChild64** [comment]
They both show 0.80.0 correctly, strange. Is it possible this is a Gentoo e-build issue?

**Comment by kcgen** [comment]
> They both show 0.80.0 correctly, strange. Is it possible this is a Gentoo e-build issue?

I'm guessing the ebuild was created in the short window of time when I had to move the 0.80.0 tag by one commit on the `release/0.80.0` branch to the next commit: https://github.com/dosbox-staging/dosbox-staging/commit/dcb4408229992328fe1326a084ca2106469898aa, which was a CI change to force the GitHub VMs to fetch all the tags to ensure the version string is based on the latest tag.

There would have only been a handful of minutes in that window of time (when I had to move the tag), however if the ebuild was created just then - then their latest tag would have been 0.80-alpha instead of 0.80.0, which is the only thing that can explain this.

Maybe they can update the ebuild against the current tag? 

Alternatively, I think most of the fixes for 0.80.1 are ready; so hopefully will soon be the latest.
